src/Tribe/PUE/Checker.php

```diff
@@ -1918,11 +1976,18 @@ public function inject_info( $result, $action = null, $args = null ) {
 		 * The callback function should take one argument - an associative array of query arguments.
 		 * It should return a modified array of query arguments.
 		 *
-		 * @uses add_filter() This method is a convenience wrapper for add_filter().
-		 *
 		 * @param callback $callback The callback function.
 		 */
 		public function add_query_arg_filter( $callback ) {
+			/**
+			 * Filters the query arguments passed to the API request.
+			 *
+			 * Plugins can hook into this filter to modify the query parameters sent to the API.
+			 *
+			 * @since TBD
+			 *
+			 * @param array $query_args The query arguments to be sent to the API.
```

Comment:
No...
`$callback`?

Reply:
Hm, is this hook working as expected then?

Reply:
@Camwyn That's a good point. I actually don't see this function being used at all anywhere. 

Looks like instead of calling the function in other spots we are using just the filter.